### PR TITLE
Fix unity settings file

### PR DESCRIPTION
**Pull request by wolf843**
test pull

### CODE REVIEW

**Comment by rajat2004** [comment]
Hey @wolf843, does this also fix the problem of an extra `settings.json` file getting created in the main directory if the `settings.json` in `Documents/AirSim` is not present?
The diff is pretty big so it's a bit difficult to follow

File: Unity/UnityDemo/Assets/AirSimAssets/Scripts/Utilities/AirSimSettings.cs [hunk 1]
```diff
@@ -37,287 +38,3753 @@ public struct CamerasSettings {
...
+                Debug.LogError("Error: 'settings.json' file SettingsVersion missing!  SettingsVersion attribute requried.");
+#if UNITY_EDITOR
+                EditorUtility.DisplayDialog("'settings.json' file SettingsVersion Error!!!",
+                    "'settings.json' file is missing or SettingsVersion", "Exit");
+#endif
+                Application.Quit();
+            }
+
+            // Verify SettingsVersion is greater than 1.2
+            if (SettingsVersion != 1.2)
```
Comment (rajat2004) [inline Unity/UnityDemo/Assets/AirSimAssets/Scripts/Utilities/AirSimSettings.cs]: Should be `<` here

**Comment by rajat2004** [comment]
> Hey @wolf843, does this also fix the problem of an extra `settings.json` file getting created in the main directory if the `settings.json` in `Documents/AirSim` is not present?

I think the problem still persists, since the [GetAirSimSettingsFileName()](https://github.com/microsoft/AirSim/blob/50b9979a5ed44664f2b6032999014954359d5988/Unity/UnityDemo/Assets/AirSimAssets/Scripts/Utilities/AirSimSettings.cs#L2329-L2357) is the same as in the [older InitializeAirsim.cs](https://github.com/microsoft/AirSim/blob/master/Unity/UnityDemo/Assets/AirSimAssets/Scripts/InitializeAirSim.cs#L42-L67)

Could be mistaken, please correct me if that's the case

**Comment by rajat2004** [comment]
@madratman I think the best way to get the settings fixed would be to do piecewise PRs or atleast smaller commits, will make it much easier to review and test

**Comment by zimmy87** [comment]
closed due to license agreement not being signed and unanswered old CR feedback

**Pull request closed without merging**